index.html

```diff
@@ -800,12 +800,12 @@ <h4><dfn>Update the EditContext</dfn></h4>
                     </ol>
                 </li>
                 <li>If |editContext|'s [=is composing=] is true, then:
-                    <ol>Let |insertionStart| be |replacementRangeStart|.</ol>
-                    <ol>Let |insertionEnd| be |replacementRangeEnd|.</ol>
+                    <ul><li>Let |insertionStart| be |replacementRangeStart|.</li></ul>
```

Comment:
For consistency with the rest of the document these substeps should be ordered. Could we do it this way instead?

```
                <li>If |editContext|'s [=is composing=] is true, then:
                    <ol>
                        <li>Let |insertionStart| be |replacementRangeStart|.</li>
                        <li>Let |insertionEnd| be |replacementRangeEnd|.</li>
                    </ol>
                </li>
                <li>Otherwise:
                    <ol>
                        <li>Let |insertionStart| be |selectionStart|.</li>
                        <li>Let |insertionEnd| be |selectionEnd|.</li>
                    </ol>
                </li>
```

Reply:
Make sense to me! I've made the update. Thanks.

Reply:
Thank you!